### PR TITLE
Documentación para la generación de ficheros CNMC SIPS

### PR DESCRIPTION
**Pull request by polsala**
### Description:

Adds new documentation for SIPS generation on distri/adm_pub/cnmc

### Tasks:

- [x] Add Catalan documentation for CNMC SIPS generation on electric power distributors
- [x] Add images
- [x] Spanish translation (waiting for review)

### Links:

* [cnmc.md #Generació de fitxers oficials CNMC SIPS](http://builds.gisce.net/powerp-docs/powerp_docu_sips_3_0_distri_generate_official_spreadsheet/ca/distri/adm-pub/cnmc/#generacio-de-fitxers-oficials-cnmc-sips)
* [cnmc.md #Generación de ficheros oficiales CNMC SIPS](http://builds.gisce.net/powerp-docs/powerp_docu_sips_3_0_distri_generate_official_spreadsheet/es/distri/adm-pub/cnmc/#generacion-de-ficheros-oficiales-cnmc-sips)

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_sips_3_0_distri_generate_official_spreadsheet/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_sips_3_0_distri_generate_official_spreadsheet/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by polsala** [comment]
@jaumef I need your powers :raised_hands:

File: docs/distri/adm-pub/cnmc.md [hunk 1]
```diff
@@ -143,3 +143,39 @@ Camp a recalcular| Descripció                                        | Longitud
...
 CINI             | Codi d'identificació normalitzada d'instal·lacions|   8     | cadena
 TI               | Tipus d'instalació de la CNMC                     |   -     | cadena
+
+
+## Generació de fitxer oficials CNMC SIPS
+
+Segons la llei 24/2013 del 26 de desembre i juntament amb el 3r article del
+Reial decret 1011/2009 del 19 de juny, amb la finalitat de tenir una base de
+dades actualitzada de consumidors i punts de subministrament la CNMC requereix
+de l'enviament d'una serie de fitxer per part de les distribuïdores.
```
Comment (MiquelIR) [inline docs/distri/adm-pub/cnmc.md]: d'una serie de fitxer
d'una serie de fitxers

File: docs/distri/adm-pub/cnmc.md [hunk 2]
```diff
@@ -143,3 +143,39 @@ Camp a recalcular| Descripció                                        | Longitud
...
+
+### Tipus de fitxers CNMC SIPS
+
+**Consums:** Fitxer filtrat per codis **CUPS** amb els consums efectuats
+separats per periodes.
+
+**Punts de subministrament:** Fitxer filtrat per codis **CUPS** amb informació
+sobre els punts de subministrament donats d'alta.
+
```
Comment (MiquelIR) [inline docs/distri/adm-pub/cnmc.md]: It will look better if you put them as a list

File: docs/distri/adm-pub/cnmc.md [hunk 3]
```diff
@@ -143,3 +143,39 @@ Camp a recalcular| Descripció                                        | Longitud
...
+L'assistent facilita la generació automàtica d'un fitxer **ZIP** amb els dos
+fitxers especificats anteriorment.
+
+Per localitzar l'assistent es pot fer a
+**Administració pública > CNMC > SIPS > Exportación fichero CNMC SIPS**
+
+![](../_static/cnmc/sips_1.png)
+
+Un cop obert, obrirà una finestra amb el botó **Exportar**. En premer el botó automaticament generarà el fitxer **ZIP** i canviara de finestre.
```
Comment (MiquelIR) [inline docs/distri/adm-pub/cnmc.md]: A lot of mistakes in this phrase. Review it please

File: docs/distri/adm-pub/cnmc.md [hunk 4]
```diff
@@ -143,3 +143,39 @@ Camp a recalcular| Descripció                                        | Longitud
...
+
+![](../_static/cnmc/sips_2.png)
+
+![](../_static/cnmc/sips_3.png)
+
+Un cop a la darrera finestra, amb el botó **Abrir** accedira directament al contingut del **ZIP** generat.
```
Comment (MiquelIR) [inline docs/distri/adm-pub/cnmc.md]: Write the button name with the same language as the docs please XD

Comment (polsala) [reply docs/distri/adm-pub/cnmc.md]: Okaay :rofl:

**Comment by gdalmau** [comment]
@polsala You can copy and paste your text before committing in a corrector (e.g. https://www.softcatala.org/corrector/) if you need it.

File: docs/distri/adm-pub/cnmc.md (hunk 1 above)
Comment (jaumef) [inline docs/distri/adm-pub/cnmc.md]: `serie` → `sèrie`

File: docs/distri/adm-pub/cnmc.md [hunk 5]
```diff
@@ -143,3 +143,39 @@ Camp a recalcular| Descripció                                        | Longitud
 :----------------|:--------------------------------------------------|:-------:|:------
 CINI             | Codi d'identificació normalitzada d'instal·lacions|   8     | cadena
 TI               | Tipus d'instalació de la CNMC                     |   -     | cadena
+
+
+## Generació de fitxer oficials CNMC SIPS
+
+Segons la llei 24/2013 del 26 de desembre i juntament amb el 3r article del
+Reial decret 1011/2009 del 19 de juny, amb la finalitat de tenir una base de
```
Comment (jaumef) [inline docs/distri/adm-pub/cnmc.md]: Si hi ha un link a un BOE on s'informa d'això s'hauria d'afegir... 
- Real decreto 1011/2009 en BOE "BOE-A-2009-10220"(https://www.boe.es/boe/dias/2009/06/20/pdfs/BOE-A-2009-10220.pdf)
- Lei 24/2013 en BOE "BOE-A-2013-13645" (https://www.boe.es/boe/dias/2013/12/27/pdfs/BOE-A-2013-13645.pdf)

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_sips_3_0_distri_generate_official_spreadsheet/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/distri/adm-pub/cnmc.md (hunk 3 above)
Comment (jaumef) [inline docs/distri/adm-pub/cnmc.md]: I'd change the build of the sentence (without remarking typos):

`En premer el botó automaticament generarà el fitxer i canviara de finestre ` → `Prement el botó es genera el fitxer`

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_sips_3_0_distri_generate_official_spreadsheet/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_sips_3_0_distri_generate_official_spreadsheet/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by jaumef** [comment]
@polsala Ready for translation :+1: 
Remember to merge from master before to avoid conflicts!

File: docs/distri/adm-pub/cnmc.md [hunk 6]
```diff
@@ -143,3 +143,39 @@ Camp a recalcular| Descripció                                        | Longitud
...
+
+![](../_static/cnmc/sips_1.png)
+
+L'assistent obra una finestra amb el botó **Exportar**. Prement el botó es genera el fitxer **ZIP**.
+
+![](../_static/cnmc/sips_2.png)
+
+![](../_static/cnmc/sips_3.png)
+
+Un cop a la darrera finestra, amb el botó **Obrir** accedira directament al contingut del **ZIP** generat.
```
[the remark this answers is not in the document]
Comment (jaumef) [reply docs/distri/adm-pub/cnmc.md]: De fet millor canviar la conjugació, `accedirà`→`s'accedeix`

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_sips_3_0_distri_generate_official_spreadsheet/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_sips_3_0_distri_generate_official_spreadsheet/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_sips_3_0_distri_generate_official_spreadsheet/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_sips_3_0_distri_generate_official_spreadsheet/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by gdalmau** [comment]
@polsala Use for example https://www.correctorortografico.com/ as there are words that don't exist in spanish translation.

File: docs/distri/adm-pub/cnmc.md [hunk 7]
```diff
@@ -170,12 +170,12 @@ Per localitzar l'assistent es pot fer a
...
 
-L'assistent obra una finestra amb el botó **Exportar**. Prement el botó es genera el fitxer **ZIP**.
+L'assistent obre una finestra amb el botó **Exportar**. Prement el botó es genera el fitxer **ZIP**.
 
 ![](../_static/cnmc/sips_2.png)
 
 ![](../_static/cnmc/sips_3.png)
 
-Un cop a la darrera finestra, amb el botó **Obrir** accedira directament al contingut del **ZIP** generat.
+Un cop a la darrera finestra, amb el botó **Obrir** s'accedeix directament al contingut del **ZIP** generat.
```
Comment (jaumef) [inline docs/distri/adm-pub/cnmc.md]: `s'accedeix directament al contingut del **ZIP** generat` → `s'accedeix al contingut del **ZIP** generat`

S'accedeix ~directament~
S'accedeix ~automàticament~

File: docs/distri/adm-pub/cnmc.md [hunk 8]
```diff
@@ -143,3 +143,39 @@ Camp a recalcular| Descripció                                        | Longitud
...
+
+![](../_static/cnmc/sips_1.png)
+
+L'assistent obre una finestra amb el botó **Exportar**. Prement el botó es genera el fitxer **ZIP**.
+
+![](../_static/cnmc/sips_2.png)
+
+![](../_static/cnmc/sips_3.png)
+
+Un cop a la darrera finestra, amb el botó **Obrir** s'accedeix directament al contingut del **ZIP** generat.
```
Comment (jaumef) [inline docs/distri/adm-pub/cnmc.md]: `s'accedeix directament al contingut del **ZIP** generat` → `s'accedeix al contingut del **ZIP** generat`

S'accedeix ~directament~
S'accedeix ~automàticament~

File: docs/distri/adm-pub/cnmc.md [hunk 9]
```diff
@@ -143,3 +143,39 @@ Camp a recalcular| Descripció                                        | Longitud
...
+
+- **Consums:** Fitxer filtrat per codis **CUPS** amb els consums efectuats
+separats per períodes.
+
+- **Punts de subministrament:** Fitxer filtrat per codis **CUPS** amb informació
+sobre els punts de subministrament donats d'alta.
+
+### Assistent per a la generació de fitxers CNMC SIPS
+
+L'assistent facilita la generació automàtica d'un fitxer **ZIP** amb els dos
```
Comment (jaumef) [inline docs/distri/adm-pub/cnmc.md]: `L'assistent facilita la generació automàtica d'un fitxer **ZIP** amb els dos` → `L'assistent facilita la generació d'un fitxer **ZIP** amb els dos`

S'accedeix ~directament~
S'accedeix ~automàticament~

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_sips_3_0_distri_generate_official_spreadsheet/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/distri/adm-pub/cnmc.md [hunk 10]
```diff
@@ -143,3 +143,39 @@ Camp a recalcular| Descripció                                        | Longitud
...
+- **Punts de subministrament:** Fitxer filtrat per codis **CUPS** amb informació
+sobre els punts de subministrament donats d'alta.
+
+### Assistent per a la generació de fitxers CNMC SIPS
+
+L'assistent facilita la generació d'un fitxer **ZIP** amb els dos
+fitxers especificats anteriorment.
+
+Per localitzar l'assistent es pot fer a
+**Administració pública > CNMC > SIPS > Exportación fichero CNMC SIPS**
```
Comment (jaumef) [inline docs/distri/adm-pub/cnmc.md]: La ruta hauria d'estar totalment en català a no ser que l'assistent també tingui el nom en castellà al ERP `Exportación fichero CNMC SIPS`

**Pull request merged**